### ISSUE TITLE
Windows 11, Strawberry Perl 5.32 64 bit. Install fails

### ISSUE THREAD

**Issue by GrandfatherJ** [Corion/WWW-Mechanize-Chrome#65]
cpanm --force gets the job done. Haven't tested the result yet, but will update this if I find issues over the next few days.
[build.log](https://github.com/Corion/WWW-Mechanize-Chrome/files/8695373/build.log)

**Comment by Corion** [comment]
Yes - the error is "merely" that there is a timing/understanding issue between the tests and Chrome and sometimes the id of a node cannot be retrieved as it is zero.

I don't know what causes this, but it is a known issue and usually temporary.

**Comment by GrandfatherJ** [comment]
if it's timing, could a wait and retry fix it?
